packages/angular/src/generators/component-test/schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "NxAngularComponentTestGenerator",
+  "type": "object",
+  "cli": "nx",
+  "description": "Create a `*.cy.ts` file for Cypress component testing for an Angular component.",
+  "properties": {
+    "projectPath": {
+      "type": "string",
+      "description": "The path to the project.",
+      "examples": ["libs/my-fancy-ng-project"]
```

Comment:
Are we able to ask the user to provide the name of the project instead of the path to it and then use the project's configuration to find the root?

```ts
const projectPath = readProjectConfiguration(tree, options.project).root;
```

Reply:
If we can, then we can add `"x-dropdown": "projects"` to the schema for this property which allows Nx Console to display a dropdown of Projects in the Workspace to choose from

Reply:
I agree, it would be easier for the user!